pkg/cmd/plugincheck2/main_test.go

```diff
@@ -28,6 +28,12 @@ type JsonReport struct {
 	PluginValidator map[string][]Issue `json:"plugin-validator"`
 }
 
+type TC struct {
+	File       string
+	ExtraArgs  string
+	JsonReport JsonReport
+}
```

Comment:
Nit: can we unexport this?

```suggestion
type tc struct {
	file       string
	extraArgs  string
	jsonReport JsonReport
}
```